test/cfg/windows.cpp

```diff
@@ -960,27 +957,27 @@ HANDLE test_CreateThread(LPSECURITY_ATTRIBUTES lpThreadAttributes,
 
     // cppcheck-suppress leakReturnValNotUsed
     // cppcheck-suppress uninitvar
-    (void) CreateThread(lpThreadAttributes, dwStackSize, lpStartAddress, lpParameter, uninit_dwCreationFlags, lpThreadId);
```

Comment:
I wonder is there a leakReturnValNotUsed even with the void cast? The void cast should not prevent that imho.

Reply:
Well, that's what the ticket is about. Someone wanted to suppress the warning with `(void)`...